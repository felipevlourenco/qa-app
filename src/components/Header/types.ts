import { ReactNode } from 'react'

export interface HeaderProp {
  children: ReactNode
}
