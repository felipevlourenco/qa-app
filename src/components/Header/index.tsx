import { HeaderProp } from 'components/Header/types'
import styles from 'components/Header/styles.module.css'

const Header = ({ children }: HeaderProp) => {
  return <header className={styles.header}>{children}</header>
}

export default Header
