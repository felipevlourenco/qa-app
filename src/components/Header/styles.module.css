.header {
  position: fixed;
  top: 0;
  width: 100%;
  height: 4rem;
  font-size: 1.8rem;
  display: flex;
  justify-content: center;
  align-items: center;
}