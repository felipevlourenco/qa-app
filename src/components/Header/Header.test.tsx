import { render } from '@testing-library/react'
import Header from 'components/Header'

describe('components/Header', () => {
  it('should render', () => {
    const { container } = render(<Header>Header title</Header>)
    expect(container).toMatchSnapshot()
  })
})
