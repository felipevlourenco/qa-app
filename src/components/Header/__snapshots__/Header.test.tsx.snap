// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Header should render 1`] = `
<div>
  <header
    class="header"
  >
    Header title
  </header>
</div>
`;
